package.json

```diff
@@ -37,8 +37,8 @@
   },
   "scripts": {
     "build": "./node_modules/node-sass/bin/node-sass app.scss dist/app.min.css --style compressed",
-    "install": "npm run build",
+    "postinstall": "npm run build",
```

Comment:
It seems somehow strange to replace `install`, as this suggests we don't mean to download dependencies - would `postinstall` be more appropriate?

Reply:
I'm not sure `postinstall` is a good idea either.  The `install` hook is run when you install the project for local dev, but also when installing the project as a dependency.  This just breaks for me when used in Shipshape.

What's the rationale behind adding the hook?